interfaces/insight.ts

```diff
@@ -1,16 +1,20 @@
-import {FiledRequest} from "./filedRequest";
-import {User} from "./user";
 
 export const INSIGHTS_COLLECTION = "insights"
 
 export interface Insight {
     createdBy: string; // userId
     createDate: number;
+    lastUpdated: number;
+    ticketId: string,
     title: string;
-    request: string[]; // filedRequestIds
+    imgs: {src: string, alt: string}[],
+    outcomeNumber: number,
+    keyStat: string,
+    keyNumber: string,
+    tldr: string,
+    requests: string[]; // filedRequestIds
     collaborators: string[] // userIds/groupsIds;
-    description?: string;
-    attachments?: string[]; // TODO Support complex media types
+    takeaways?: string;
```

Comment:
if takeaways are plural it might make sense to make this an array?

Reply:
Changed to takeaway singular just so there's no ambiguity. Given that multiple insights can stem from a FiledRequest, I'd say each Insight should have a singular theme, can revisit this if we want to expand it